<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgIntercom</class>
 <widget class="QDialog" name="DlgIntercom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>List of Intercoms</string>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblState">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Select to Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbExtn1">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn1</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn2</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn3</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn4</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pbExtn5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn5</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn6</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn7</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn8">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Extn8</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pbExtn9">
        <property name="text">
         <string>Extn9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn10">
        <property name="text">
         <string>Extn10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn11">
        <property name="text">
         <string>Extn11</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn12">
        <property name="text">
         <string>Extn12</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pbExtn13">
        <property name="text">
         <string>Extn13</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn14">
        <property name="text">
         <string>Extn14</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn15">
        <property name="text">
         <string>Extn15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtn16">
        <property name="text">
         <string>Extn16</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgIntercom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
